Dockerfile

```diff
@@ -31,5 +32,6 @@ VOLUME /config /data
 EXPOSE 5232
 
 COPY docker-entrypoint.sh /usr/local/bin
+RUN chmod 555 /usr/local/bin/docker-entrypoint.sh
```

Comment:
why is this needed (because the script is already executable in the repo, no?)

Reply:
Yeah can definitely black box test radicale. 

The reason for the permission check is that by default (when using `COPY`) the file has write permissions (755), although oddly it differed between running it locally and on travis (775 vs 755). So the thinking was to be explicit and set read/execute permissions only as that's all that is needed.

Granted it's moot if running your container as `--read-only` though